Makefile

```diff
@@ -10,7 +10,7 @@ DOCKER_LOGIN_ENABLED ?= true
 
 ARTIFACTS := _out
 TOOLS ?= ghcr.io/talos-systems/tools:v0.4.0
-PKGS ?= v0.4.1
+PKGS ?= v0.5.0-alpha.0
```

Comment:
this won't work, but we have bump here: https://github.com/talos-systems/talos/pull/3285

Reply:
yep, just rebased it